persistence/scaladsl/src/main/scala/com/lightbend/lagom/scaladsl/persistence/EventStreamElement.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package com.lightbend.lagom.scaladsl.persistence
+
+/**
+ * Envelope for events in the eventstream, provides additional data to the actual event
+ */
+final class EventStreamElement[+Event](
```

Comment:
I guess we don't want to use a `case class` to make it easier to change. `unapply` would be nice though.